src/compiled-grammar/index.ts

```diff
@@ -13,6 +13,7 @@ export type CompiledSymbol =
 export interface CompiledRule {
     name : string;
     symbols : CompiledSymbol[];
+    precedence : number;
```

Comment:
Terrible attempt at implementing rule precedence (without associativity).

This helps us resolve conflicts with the following,
```
SELECT 'a' 'b'
```

The above can be interpreted in two ways,
+ `SELECT 'a' AS 'b'`
+ `SELECT 'ab'`

Reply:
MySQL grammar has lots of ambiguity.